src/utils/setup/init-stitch.js

```diff
@@ -1,10 +1,10 @@
 import { Stitch, AnonymousCredential } from 'mongodb-stitch-server-sdk';
 import { SNOOTY_STITCH_ID } from '../../build-constants';
 
-export const initStitch = async () => {
-    const stitchClient = Stitch.hasAppClient(SNOOTY_STITCH_ID)
-        ? Stitch.getAppClient(SNOOTY_STITCH_ID)
-        : Stitch.initializeAppClient(SNOOTY_STITCH_ID);
+export const initStitch = async (id = SNOOTY_STITCH_ID) => {
```

Comment:
Since this module only imports from `mongodb-stitch-server-sdk`, we can continue to use it. I parameterized the `id` here to allow it to work for both Stitch apps at build-time